waiter/src/waiter/statsd.clj

```diff
@@ -310,6 +310,13 @@
       [metric-group metric value]
       (send aggregation-agent add-value metric-group metric value gauge-metric))
 
+    (defn timing!
```

Comment:
Can we add a unit test for this?

Reply:
There's too much additional code needed for this statsd change. I'm pulling it out and I'll follow up with a separate PR once this one is merged.